content/en/security/notifications/rules.md

```diff
@@ -28,7 +28,7 @@ As you configure the rule, a preview of issues matching the notification rule co
 
 1. On the [**Notification Rules**][1] page, click **New Notification Rule**.
 2. Under **Source Types**, select the detection rule types you want to include in the notification rule.
-3. (Optional) For ASM, select the **Include Application level vulnerabilities** checkbox.
+3. (Optional) For ASM, leave the source type empty and select the **Include Application level vulnerabilities** checkbox, _or_ select the ASM Vulnerability Management source type.
```

Comment:
```suggestion
3. (Optional) For ASM, select the ASM Vulnerability Management source type. Alternatively, leave the source type empty and select the **Include Application level vulnerabilities** checkbox.
```
Seems to me that selecting the source type is the easiest option/happy path, so I think that should come first. Also, I suggest splitting the sentence into two to help with readability.

Reply:
Great point! I'll fix this so it reads better :)